### PR TITLE
Add more sharing options for content

### LINKED ISSUE #1267: Add more sharing options for content
Add more sharing options for content such as Linkedin, etc.: https://discord.com/channels/385034494555455488/445901975301914624/1076879184380579841

### PR DESCRIPTION
**Pull request by MTSaeed**
**What does this PR?**

Adds One more option of LinkedIn for sharing content.

**Steps to reproduce**

1.Try to share any post. You will see the option to share the post to LinkedIn also.

**Issue number** 

fixes #1267 

**Screenshot / Video**

https://user-images.githubusercontent.com/106739598/225836261-33aef993-76c9-45be-bf7e-0355e9fc5b37.mp4

### CODE REVIEW

**Comment by MTSaeed** [comment]
image

**Comment by MTSaeed** [comment]
image

File: src/common/helper/url-share.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import defaults from "../constants/defaults.json";
+import auth from "../pages/auth";
```
Comment (feruzm) [inline src/common/helper/url-share.ts:2]: No need this import?

File: src/common/helper/url-share.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
 
 export const makeEsteemUrl = (cat: string, author: string, permlink: string): string =>
```
Comment (feruzm) [inline src/common/helper/url-share.ts:4]: change naming to `makeEcencyUrl` instead

File: src/common/helper/url-share.ts (hunk 1 above)
Comment (MTSaeed) [reply src/common/helper/url-share.ts]: @feruzm  updated.

File: src/common/helper/url-share.ts (hunk 2 above)
Comment (MTSaeed) [reply src/common/helper/url-share.ts]: @feruzm  updated.

**Review by feruzm — APPROVED** [review]
lgtm

**Pull request merged**